modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/SegmentsExperiments.es.js

```diff
@@ -84,7 +84,11 @@ function SegmentsExperiments({
 					active={activeTab === TABS_STATES.HISTORY}
 					onClick={() => setActiveTab(TABS_STATES.HISTORY)}
 				>
+<<<<<<< HEAD
```

Comment:
Mmm, I think it can be a good idea to do an interactive rebase here so we don't add this "conflict" lines. What do you think @Stephy6 ? Maybe you can merge your last commit with this one.